packages/ui/src/views/repo/repo-branch-rules/repo-branch-settings-rules-page.tsx

```diff
@@ -111,77 +111,72 @@ export const RepoBranchSettingsRulesPage: FC<RepoBranchSettingsRulesPageProps> =
 
   return (
-    <SandboxLayout.Main>
-      <SandboxLayout.Content className="mx-auto max-w-[610px] pt-7">
-        <Text size={5} weight="medium" as="div" className="mb-10">
-          {presetRuleData ? t('views:repos.updateRule', 'Update rule') : t('views:repos.CreateRule', 'Create a rule')}
-        </Text>
-
-        <FormWrapper onSubmit={handleSubmit(onSubmit)}>
-          <BranchSettingsRuleToggleField register={register} setValue={setValue} watch={watch} t={t} />
-
-          <BranchSettingsRuleNameField register={register} errors={errors} disabled={!!presetRuleData} t={t} />
-
-          <BranchSettingsRuleDescriptionField register={register} errors={errors} t={t} />
-
-          <BranchSettingsRuleTargetPatternsField
-            watch={watch}
-            setValue={setValue}
-            register={register}
-            errors={errors}
-            t={t}
-          />
-
-          <BranchSettingsRuleBypassListField
-            register={register}
-            errors={errors}
-            setValue={setValue}
-            watch={watch}
-            bypassOptions={principals}
-            t={t}
-            setPrincipalsSearchQuery={setPrincipalsSearchQuery}
-            principalsSearchQuery={principalsSearchQuery}
-          />
-
-          <BranchSettingsRuleListField
-            rules={rules}
-            recentStatusChecks={recentStatusChecks}
-            handleCheckboxChange={handleCheckboxChange}
-            handleSubmenuChange={handleSubmenuChange}
-            handleSelectChangeForRule={handleSelectChangeForRule}
-            handleInputChange={handleInputChange}
-            t={t}
-          />
-
-          <Fieldset className="mt-5">
-            <ControlGroup>
-              <ButtonGroup>
-                <Button type="submit" disabled={!isValid || isLoading}>
-                  {!isLoading
-                    ? presetRuleData
-                      ? t('views:repos.updateRule', 'Update rule')
-                      : t('views:repos.createRuleButton', 'Create rule')
-                    : presetRuleData
-                      ? t('views:repos.updatingRule', 'Updating rule...')
-                      : t('views:repos.creatingRuleButton', 'Creating rule...')}
-                </Button>
-                <Button type="button" variant="outline">
-                  <NavLink to="..">{t('views:repos.cancel', 'Cancel')}</NavLink>
-                </Button>
-              </ButtonGroup>
-            </ControlGroup>
-          </Fieldset>
-
-          {!!apiErrorsValue && (
-            <>
-              <Spacer size={2} />
-              <Text size={1} className="text-destructive">
-                {apiErrorsValue}
-              </Text>
-            </>
-          )}
-        </FormWrapper>
-      </SandboxLayout.Content>
-    </SandboxLayout.Main>
+    <SandboxLayout.Content className="max-w-[570px] px-0">
+      <Text size={5} weight="medium" as="div" className="mb-10">
```

Comment:
```suggestion
      <h1 className="mb-10 font-medium text-24">
```
Keep in mind that we should avoid using the `Text` component whenever possible, as it is planned to be removed in the future. Instead, we can use the appropriate HTML tag directly.

Also, since this serves as the page title, I would consider using an `h1` tag instead of a `div`.